src/layers/object.rs

```diff
@@ -45,6 +44,13 @@ impl ObjectLayerData {
         });
         Ok((ObjectLayerData { objects, colour: c }, properties))
     }
+
+    /// Returns the data belonging to the objects contained within the layer, in no particular
+    /// order.
```

Comment:
Why would this be "in no particular order", whereas `objects()` will iterate "in the order they were declared in the TMX file"? The order is the same, right?

The order in the TMX file is relevant, btw. It is the rendering order, for objects layers with `draworder` set to `index`, and it can then be manually controlled in Tiled. But I'm not sure how much of this needs to be documented here.